code/game/objects/items/weapons/flamethrower.dm

```diff
@@ -144,7 +144,8 @@
 
 
 //Called from turf.dm turf/dblclick
-/obj/item/weapon/flamethrower/proc/flame_turf(turflist)
+/obj/item/weapon/flamethrower/proc/flame_turf(var/list/turflist)
```

Comment:
These kind of unrelated changes should really be made into a separate pr next time.

Reply:
I reworked flamethrowers as a major balancing component of the new blob and spacevines